src/models/chan.js

```diff
@@ -23,18 +23,21 @@ function Chan(attr) {
 	}, attr));
 }
 
-Chan.prototype.sortUsers = function() {
-	this.users = _.sortBy(
-		this.users,
-		function(u) { return u.name.toLowerCase(); }
-	);
-
-	["+", "%", "@", "&", "~"].forEach(function(mode) {
-		this.users = _.remove(
-			this.users,
-			function(u) { return u.mode === mode; }
-		).concat(this.users);
-	}, this);
+Chan.prototype.sortUsers = function(irc) {
+	var userModeSortPriority = {};
+	irc.network.options.PREFIX.forEach(function(prefix, index) {
```

Comment:
> I didn't want to pipe the whole irc-fw options/networks objects to the client, as they have a lot of data.

Yep, agreed!

However, I think those add some confusion to the codebase overall, I'm sure we can encapsulate this nicely in the models and interface that (`options`, `serverOptions`, ...) via methods. What do you think? I can try to spend some time on that if that's fine by you.